_posts/2024-06-28-this-week-in-rails.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Immutable Rails Cache Header, Local Assigns in Strict Local Templates, Threaded Active Storage Mirror Service"
+categories: news
+author: zzak
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2024-06-28
+---
+
+
+Hi, it's [zzak](https://github.com/zzak). Let's explore this week's changes in the Rails codebase.
+
+
+[Support "immutable" directive in Cache-Control](https://github.com/rails/rails/pull/52197)
+
```

Comment:
Shouldn't we use the double space at the end of the previous line so we don't get a new paragraph, just a line break?